### ISSUE TITLE
url_encode uses '+' instead of '%20'

### ISSUE THREAD

**Issue by Rio517** [colinta/SublimeStringEncode#25]
I noticed that for the past year or so the url_encode function of the package tends to encode URLs with a plus sign instead of the more safe %20.  Is there a chance we can use %20 or enable an option for it?

Thanks!

**Comment by colinta** [comment]
I like that idea!

**Comment by colinta** [comment]
Available in 2.1.0 as the `old_school` argument.  Defaults to `True`, for backwards compatibility, but set it to `False` to get your desired behavior.  Let me know if there are more characters like this (or submit a PR :smiley:).

```
{ "keys": ["super+shift+5"], "command": "url_encode", "args": {"old_school": false} }
```

**Comment by Rio517** [comment]
Wow. Thanks man!  :)
